proposals/lip-0028.md

```diff
@@ -31,37 +31,37 @@ When a transaction object is serialized, also the asset property has to be seria
 
-For this reason, this LIP proposes to divide the serialization-deserialization process. First it defines a serialization-deserialization process for the whole transaction object with the `baseTransaction` JSON schema where the `asset` property is assumed to be of type `bytes`. Secondly, it defines the serialization-deserialization process for the `asset` property with the corresponding `asset` property JSON schema depending on the type of the transaction (specified in the `type` property). There are 7 different `asset` property schemas, one for each of the transaction types in the Lisk protocol. In order to simplify the creation of custom transactions with the SDK and to make the validation process less error prone, every property defined inside a given `asset` schema is required.
+For this reason, this LIP proposes to divide the serialization-deserialization process. First it defines a serialization-deserialization process for the whole transaction object with the `transactionSchema` JSON schema where the `asset` property is assumed to be of type `bytes`. Secondly, it defines the serialization-deserialization process for the `asset` property with the corresponding `asset` property JSON schema depending on the type of the transaction (specified by the unique combination of `moduleID` and `assetID` properties). There are 7 different `asset` property schemas, one for each of the transaction types in the Lisk protocol. In order to simplify the creation of custom transactions with the SDK and to make the validation process less error prone, every property defined inside a given `asset` schema is required.
 
 ## Specification
 
-Assuming that the JSON schemas for the `baseTransaction` and for the `asset` properties are provided according to the specifications in the [LIP 0027](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0027.md), the serialization and deserialization of a transaction object are done as follows.
+Assuming that the JSON schemas for the `transactionSchema` and for the `asset` properties are provided according to the specifications in the [LIP 0027](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0027.md), the serialization and deserialization of a transaction object are done as follows.
 
 ### Serialization
 
 Consider a data structure `trsData` representing a valid (signed or unsigned) transaction object to be serialized. The serialization procedure is done in 3 steps:
 
-1. The `trsData.asset` property is serialized to bytes according to the corresponding  `asset` property schema specified by the `trsData.type` property (see [below](#baseTransaction-schema)).
+1. The `trsData.asset` property is serialized to bytes according to the corresponding  `asset` property schema specified by the combination of `trsData.moduleID` and `trsData.assetID` properties (see [below](#transactionSchema-schema)) as specified in [LIP 0033](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0033.md#specification).
```

Comment:
```suggestion
1. The `trsData.asset` property is serialized to bytes according to the corresponding  `asset` property schema given by the values of `trsData.moduleID` and `trsData.assetID`. The correspondence between `asset` property schema and values of `trsData.moduleID` and `trsData.assetID` is defined in [LIP 0036](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0036.md#specification).
```

Reply:
Both "see below" and "as specified in" together are a bit confusing without saying what is defined where. I already changed the link although it is currently not working.

Reply:
fixed